index.mjs

```diff
@@ -176,6 +176,7 @@ export {
     BooleanAnd,
     BooleanNot,
     BooleanOr,
+    checkColorGreen,
```

Comment:
The name of the class should follow the naming conventions and start by a capital C

Reply:
okay, i will change it